### PR TITLE
Example/dependency injection

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**

### CODE REVIEW

File: src/Orbit.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Schema } from '@orbit/data'
+import Store from '@orbit/store'
+import JSONAPISource from '@orbit/jsonapi'
+import Coordinator, { RequestStrategy, SyncStrategy, EventLoggingStrategy, LogLevel } from '@orbit/coordinator'
+
+export default class Orbit {
+  public currentSchema: any
```
Comment (tniezg) [inline src/Orbit.ts]: we may want to sent `readonly` on some of these, or use getters

Comment (igorbezsmertnyi) [reply src/Orbit.ts]: @tniezg I will be preparing interfaces for each variable. It has `readonly` method inside https://basarat.gitbooks.io/typescript/docs/types/readonly.html

File: webpack.config.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   ],
   entry: {
     index: [
-      resolve(baseDirectoryPath, 'load-sourcemaps.js'),
+      // resolve(baseDirectoryPath, 'load-sourcemaps.js'),
```
Comment (tniezg) [inline webpack.config.js]: Accidental comment?

File: src/schema/index.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { product } from './prorduct'
```
Comment (tniezg) [inline src/schema/index.ts]: `prorduct` - typo

**Pull request merged**